bbcode-src/tags/img.js

```diff
@@ -0,0 +1,12 @@
+import { toNode } from "../utils/common";
+
+    /**
+     * @file Adds [img] to bbcode
+     * @example [img]imgURL[/img]
+     */
+    export const img = (node) => {
+        const imgContent = Object.values(node.content).join("");
+        const imgSrcParts = imgContent.split('/');
+        const imgHTML = `<img src="${imgContent}" alt="${imgSrcParts[imgSrcParts.length - 1]}" data-zoom-target="1" loading="lazy"/>`;
+        return toNode("div", { class: "bb-height-restrict" }, imgHTML);
```

Comment:
this is returning div.bb-height-restrict containing image. It should return just the image node.

Reply:
Returning without styling did not allow the image to appear. I have applied some sass just for images which repeats height-restrict's img design.